marlowe-actus/src/Language/Marlowe/ACTUS/Model/SCHED/ContractSchedule.hs

```diff
@@ -10,10 +10,11 @@ import           Language.Marlowe.ACTUS.Model.SCHED.ContractScheduleModel (_SCHE
                                                                            _SCHED_MD_PAM, _SCHED_PP_PAM, _SCHED_PRD_PAM,
                                                                            _SCHED_PY_PAM, _SCHED_RRF_PAM, _SCHED_RR_PAM,
                                                                            _SCHED_SC_PAM, _SCHED_TD_PAM)
+import           Data.Set                                                 (fromList, toAscList)
 
 
 schedule :: EventType -> ContractTerms -> Maybe [ShiftedDay]
-schedule ev ContractTerms {..} = case contractType of
+schedule ev ContractTerms {..} = toAscList <$> fromList <$> case contractType of
```

Comment:
`toAscList . fromList <$>` no?

Reply:
okay, why not

Reply:
Come on, split this into sub-expressions. It not a contest of one-liners :)

Reply:
for me it's about 2 things
1. what does 'normal' haskell code style suggest
2. I feel like you should only map one function over something

but it's just a lint, not really important at all.

Reply:
lol, I'll just delete it - I sort it in sampleCashFlows anyways